CHANGELOG.md

```diff
@@ -66,6 +67,7 @@
 - Improve error logging in WorkflowStateProcessor.
 - Replace FindBugs with SpotBugs.
 - Drop index from main table that was only used for archiving purposes.
+- Increase workflow and action identifiers in code to 64 bits for future proofing. The SQL table definitions stay the same for now, but no code changes are needed if database is altered to use bigint identifiers in large systems. The default 31 bits should be enough for 10 workflows per second with each having 20 steps for 8 years constant load.
```

Comment:
Here the caveat is that multi-master databases jump over a lot of indices. In our galera the gap is 3, so 1, 4, 7, 10...